website/client/src/components/header/menu.vue

```diff
@@ -4,6 +4,7 @@
     <profileModal />
     <report-flag-modal />
     <send-gems-modal />
+    <select-user-modal />
```

Comment:
given that this modal seems specific for G1G1 maybe we shouldn't load it as long as it's not active?

Reply:
Good call!

Reply:
...although see https://github.com/HabitRPG/habitica/pull/11823/files for possible non-G1G1 usage

Reply:
in that case nevermind!